robot/Crisis-Management/resources/CommunityHomePageObject.py

```diff
@@ -125,4 +125,14 @@ def verify_user_details_on_community_home_tab(self,**kwargs):
                 locator = cm_lex_locators["community_home_locators"]["staff_info"].format(value)
                 self.selenium.wait_until_page_contains_element(locator,
                 error=f"'{key}' with the '{value}' is not available in the community page")
-        
\ No newline at end of file
+
+
+    def click_user_info(self,name,value):
+        """ Click on user name on the Community Home page
+        """
+        locator = cm_lex_locators["community_home_locators"]["user_info"].format(name)
+        field = self.selenium.get_webelement(locator)
+        self.selenium.click_element(field)
+        locator = cm_lex_locators["community_home_locators"]["user_sub_info"].format(value)
+        field = self.selenium.get_webelement(locator)
+        self.selenium.click_element(field)
```

Comment:
After this click statement, you can validate whether the following:
`self.selenium.page_should_contain("COVID19 Division")
  self.selenium.page_should_contain("Robot Hospital")
`

Reply:
Ok, Thanks !!

Reply:
Done